apps/ai-backoffice/components/ui/use-toast.ts

```
@@ -0,0 +1,189 @@
+
+  return {
+    id: id,
+    dismiss,
+    update,
+  }
+}
+
+function useToast() {
+  const [state, setState] = React.useState<State>(memoryState)
+
+  React.useEffect(() => {
```

Biome

[error] This hook specifies more dependencies than necessary: state

This dependency can be removed from the list.


(lint/correctness/useExhaustiveDependencies)